config/xml_schemas/testlist.xsd

```diff
@@ -16,7 +16,7 @@
         <xs:element minOccurs="1" maxOccurs="1" ref="machines"/>
         <xs:element minOccurs="0" ref="options"/>
       </xs:sequence>
-      <xs:attribute name="compset" use="required" type="xs:NCName"/>
+      <xs:attribute name="compset" use="required" type="xs:string"/>
```

Comment:
That this change was required leads me to think that perhaps some new compset name violates the naming convention - can you explain?

Reply:
This was needed in order to have the long case name in the test - which is what was needed for the aux_cdeps tests.
If I recall - you pointed out that we needed to change this.